components/SideBar.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { SVGProps } from 'react';
+import { useSidebarContext } from './context';
+import Link from 'next/link';
+import {
+  LogoutIcon,
+  EditIcon,
+  LocationIcon,
+  DashboardIcon,
+  ExitIcon,
+} from './icons/sidebar_icons';
+import { LinkProps } from 'next/link';
+
+type MenuLinkProps = {
+  linkProps: LinkProps;
+  label: string;
+  svg: SVGProps<SVGElement>;
+  textColor: string;
+};
+
+const MenuLink: React.FC<MenuLinkProps> = ({
+  linkProps,
+  label,
+  svg,
+  textColor,
+}) => {
+  return (
+    <>
+      <Link {...linkProps}>
+        <div className="mb-5 px-4 py-2 flex items-center rounded-md text-gray-600 hover:text-gray-700 hover:bg-ocf-gray-1000 transition-colors transform">
+          <>{svg}</>
+          <span className={`mx-4 font-medium flex-1 align-center ${textColor}`}>
+            {label}
+          </span>
+        </div>
+      </Link>
+    </>
```

Comment:
Since this component's root element is a fragment, and the fragment only has one child, the fragment can be removed.

```suggestion
      <Link {...linkProps}>
        <div className="mb-5 px-4 py-2 flex items-center rounded-md text-gray-600 hover:text-gray-700 hover:bg-ocf-gray-1000 transition-colors transform">
          <>{svg}</>
          <span className={`mx-4 font-medium flex-1 align-center ${textColor}`}>
            {label}
          </span>
        </div>
      </Link>
```